src/components/ProfileSection.js

```diff
@@ -3,6 +3,8 @@ import ListGroup from 'react-bootstrap/ListGroup';
 import { useSelector } from 'react-redux';
 
 const Profile = () => {
+  const { missions } = useSelector((state) => state.mission);
+  const Missions = missions.filter((m) => m.isMember);
   const rockets = useSelector((state) => state.rockets);
   const Rockets = rockets.filter((rock) => rock.reserved);
   console.log(Rockets);
```

Comment:
While console.log statements are great for debugging, kindly endeavour to remove them afterwards in order to clean up the code base